File: veins_test/objective-c/echo_args.m
#!/usr/bin/env -S fragletc --vein=objective-c
int main(int argc, char *argv[]) {
    int i;
    printf("Args:");
    for (i = 1; i < argc; i++)
        printf(" %s", argv[i]);
    printf("\n");
    return 0;
}

File: veins_test/fortran/echo_args.f
#!/usr/bin/env -S fragletc --vein=fortran
  integer :: n, i
  character(256) :: arg
  n = iargc()
  write(*, '(a)', advance='no') 'Args:'
  do i = 1, n
    call getarg(i, arg)
    write(*, '(a)', advance='no') ' ' // trim(arg)
  end do
  print *
